templates/base.html

```diff
@@ -77,6 +77,14 @@
                         <a href="{% url 'hc-checks' %}">Checks</a>
                     </li>
 
+                    <li {% if page == 'unresolved' %} class="active" {% endif %}>
+                        <a href="{% url 'hc-unresolved' %}">Unresolved
+                        {% if checks %}
+                        <span class="badge badge-danger">{{ num_checks }}</span>
```

Comment:
the badge should persist even when in other pages. For instance if I go back to ```My Checks```, the badge disappers